### ISSUE TITLE
WITH keyword

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/sql-parser#165]
The WITH keyword is now supported for CTEs starting with MariaDB 10.2.1 and MySQL 8.0.

See https://github.com/phpmyadmin/phpmyadmin/issues/13616

```
./lint-query --query "WITH t as (select a from t1 where b >= 'c') 
>   select * from t2,t where t2.c=t.a;"
#1: Unrecognized statement type. (near "WITH" at position 0)
```

**Comment by timwis** [comment]
Hey, any updates on this by any chance? I'm still getting phpmyadmin warnings when using `WITH`

**Comment by jucajuca** [comment]
same here... can't use WITH

**Comment by pooryadb** [comment]
any changes ?!

**Comment by ibennetch** [comment]
No, unfortunately this has not yet been added to phpMyAdmin (you can use the syntax, it just gives a warning message when you type it in).

**Comment by brunocerq** [comment]
phpMyAdmin **4.9.5**
MariaDB **10.3.23**
Still have "**1 errors were found during analysis.
Unrecognized statement type. (near "WITH" at position 0)**"

When will this be implemented?

**Comment by williamdes** [comment]
When someone will have given some of their free time to implement it ;)
Maybe you can have a try ?

**Comment by brunocerq** [comment]
William, sorry if the question looked like a "question/demand", it was not.

**Comment by williamdes** [comment]
This will be part of 5.5.0
Implemented in: #334
